bin/configure-scripts/utils.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+
+puts ""
+puts `gem install colorize`
+puts ""
+
+require "colorize"
+require "active_support"
```

Comment:
Do these indicate we actually want something like this?

```ruby
require "bundler/inline"

gemfile do
  gem "colorize"
  gem "activesupport", require: "active_support"
end
```

Reply:
Yeah, I think that's a great suggestion. Thanks!